src/components/RootNavigator.tsx

```diff
@@ -12,12 +12,14 @@ const Tab = createMaterialBottomTabNavigator();
 
 export const RootNavigator = () => {
   const lang = useTranslation();
+  const { colors } = useTheme();
+
   return (
-    <Tab.Navigator initialRouteName='Activity' shifting={true} sceneAnimationEnabled={false} labeled={false}>
-      <Tab.Screen name='Activites' component={ActivityScreen} options={{ tabBarIcon: 'plus', title: lang.navigationLabelActivites }} />
+    <Tab.Navigator initialRouteName='Activity' shifting={true} sceneAnimationEnabled={false} labeled={true} barStyle={{backgroundColor: colors.primary}} >
+      <Tab.Screen name='Activites' component={ActivityScreen} options={{ tabBarIcon: 'plus', title: lang.navigationLabelActivites  }} />
       <Tab.Screen name='Values' component={ValuesScreen} options={{ tabBarIcon: 'heart', title: lang.navigationLabelValues }} />
       <Tab.Screen name='Calendar' component={CalendarScreen} options={{ tabBarIcon: 'calendar', title: lang.navigationLabelCalendar }} />
-      <Tab.Screen name='Information' component={InformationScreen} options={{ tabBarIcon: 'information', title: lang.navigationLabelInformation }} />
+      <Tab.Screen name='Information' component={InformationScreen} options={{ tabBarIcon: 'information', title: lang.navigationLabelInformation}} />
```

Comment:
Something happened with the whitespace. I think it should be like in the unchanged rows.